### ISSUE TITLE
YAML file wont open

### ISSUE THREAD

**Issue by Dan1990bcfc** [eskay993/gamefiles#2]
Hi when I try to open the YAML video it doesn't work. Any ideas please

**Comment by eskay993** [comment]
Hi - I'll need a bit more info to help. Could you give a bit more detail about what you tried and what doesn't work?

**Comment by Dan1990bcfc** [comment]
I try to install the ywml file but when I press it it doesn't load

**Comment by eskay993** [comment]
Sorry but that's still not enough info to help.

What platform are you on (Steam Deck or regular Linux desktop)?

Which yaml file are you referring to? There are several available to download here.

How did you download the yaml file? Did you just right click on the yaml and save, or did you download a zip and extract it?

Did you follow one of my guides and if you did, which one and which step are you stuck on?

**Comment by Dan1990bcfc** [comment]
Ah sorry. I'm on steam deck. It's the yaml file for football life. I've
downloaded the game and in lutris when I go to press add script from yaml
file. I click the file and nothing happens in lutris

**Comment by eskay993** [comment]
Thanks for the info. Just to check, are you loading the yaml file or the zip file that you downloaded? If you are loading the zip file you need to extract it first to get the yaml file. The file you should load is named `sp-football-life-2023-win.yaml`.

If you are loading the yaml file, open it in a text editor like Notepad or KWrite and copy the contents and paste it here so I can check what you are loading.

**Comment by Dan1990bcfc** [comment]
thanks for the email. i'm trying to load the yaml file after extracting it.
but lutrits doesn't do anything

here is the yaml file text

game_slug: sp-football-life-2023
name: "SP Football Life 2023"
year:  2022
user: eskay993
runner: wine
slug: sp-football-life-2023-win
version: Windows
description: Installer for SP Football Life 2023 by Smokepatch
notes:
script:
  files:
  - setup: N/A:SPFL23 Setup Exe
  - update: N/A:SPFL23 Update Exe
  game:
    arch: win64
    exe: drive_c/Program Files (x86)/SP Football Life 2023/FL_2023 start.exe
    prefix: $GAMEDIR
  installer:
  - task:
      arch: win64
      description: Creating Wine prefix...
      name: create_prefix
      install_mono: false
      prefix: $GAMEDIR
  - task:
      description: |
        Installing dependencies... This may take a while...

        **** IMPORTANT ****
        Once these dependencies finish installing, the game installer will
launch. Install everything as default, EXCEPT make sure to uncheck "Start
SP Football Life 2023" at the end of the install.
      app: consolas dotnet46
      name: winetricks
      prefix: $GAMEDIR
  - task:
      description: Starting game installer...
      executable: setup
      name: wineexec
      prefix: $GAMEDIR
  - task:
      description: Starting update installer...
      executable: update
      name: wineexec
      prefix: $GAMEDIR
  - task:
      description: Running game settings...
      executable: $GAMEDIR/drive_c/Program Files (x86)/SP Football Life
2023/FL23 switcher.exe
      name: wineexec
      prefix: $GAMEDIR
  - task:
      description: Running game settings...
      executable: $GAMEDIR/drive_c/Program Files (x86)/SP Football Life
2023/Settings.exe
      name: wineexec
      prefix: $GAMEDIR
  wine:
    overrides:
      amd_ags_x64.dll: b
    version: lutris-GE-Proton7-33-x86_64

**Comment by Dan1990bcfc** [comment]
have you got facebook or anything like that as i can send a video

**Comment by eskay993** [comment]
OK the script looks correct. It looks like an issue with your Lutris install or something else on your Deck. Try updating to thelatest stable release of SteamOS if you haven't already. And also update all your apps in Desktop Mode using Discover to make sure everything is on the latest.

If you want to send a video, probably best to use YouTube or Vimeo or something like that.

**Comment by Dan1990bcfc** [comment]
Yeah ive done all that. Thank you though. I've messaged the lutris discord
in hope someone can help. Thank you for your help

**Comment by Dan1990bcfc** [comment]
I got it working. I installed the yaml through the lutris website. Thanks
for your help

**Comment by eskay993** [comment]
Great! Out of interest, what was the solution? In case others have this issue. I wasn't aware SP Football Life is available on the Lutris website.

**Comment by eskay993** [comment]
OK well glad it worked.

Actually you should really ask for permission before submitting someone's else's script. I would prefer if you didn't as I'm undecided if I do want to submit it, and even then would be better if I did so I can maintain it.

If you could remove the submission that would be appreciated.

**Comment by Dan1990bcfc** [comment]
Yeah I've removed it. Of course and I'm sorry I never asked for permission

**Comment by eskay993** [comment]
Mo worries. Thank you.
